### DISCUSSION TITLE
Corporate

### DISCUSSION THREAD

**Discussion by algarfer** [Arquisoft/wiq_es05b#40] [Ideas]
Determine a name, logo and color palette for the application.

Also any other corporate style decisions such as technology naming will be fit for this discussion.

**Comment by donatomartin** [comment]
Just decided to name the question generator service jordi lmao

**Reply by donatomartin** [reply]
Unilaterally, of course 💀💀

**Comment by donatomartin** [comment]
Just decided to name the project "Conoce y Triunfa"

**Reply by donatomartin** [reply]
Unilaterally btw 😎

**Comment by algarfer** [comment]
Color
![image](https://github.com/Arquisoft/wiq_es05b/assets/58784148/f2642eb7-48aa-4c29-b34a-83bfe3f38cb8)

**Comment by algarfer** [comment]
# First design of the app logos

## Favicon

![logo192](https://github.com/Arquisoft/wiq_es05b/assets/58784148/25d0983d-9ae4-4acf-b05e-179a6c5ade22)

## Small Logo

![logoS](https://github.com/Arquisoft/wiq_es05b/assets/58784148/6d072702-b4fe-481b-bd08-c4ce26d72f62)

## Large Logo

![logoL](https://github.com/Arquisoft/wiq_es05b/assets/58784148/f66938b8-f6bf-42f7-acd3-fd97d1472e4f)

**Comment by donatomartin** [comment]
Just changed the color palette

**Comment by donatomartin** [comment]
Bodoni MT
